burnman/solutionmodel.py

```diff
@@ -159,7 +159,41 @@ def excess_entropy(self, pressure, temperature, molar_fractions):
         """
         return 0.0
 
+class MechanicalSolution (SolutionModel):
 
+    """
+    An extremely simple class representing a mechanical solution model.
```

Comment:
can you give a definition what a mechanical solution is? Maybe in contrast to "ideal"?